packages/twenty-server/src/modules/match-participant/match-participant.service.ts

```diff
@@ -61,14 +67,16 @@ export class MatchParticipantService<
         'person',
       );
 
-    const people = await personRepository.find(
-      {
-        where: {
-          emails: Any(uniqueParticipantsHandles),
-        },
-      },
-      transactionManager,
-    );
+    const queryBuilder = buildPersonEmailQueryBuilder({
```

Comment:
I am thinking this name could be improved. the build part sounds useless since the suffix is already 'QueryBuilder' and the main interest is missing : it helps fetching person by their primary and secondary email handles. I don't have a perfect short name though.